beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel

```diff
@@ -1,3 +1,5 @@
+# gazelle:ignore
```

Comment:
why ? this would ignore the whole file

Reply:
The presence of this tag makes gazelle ignore the whole file, and not only the part just bellow this tag.
As a consequence, it's better to put this tag at the top of the file, so the reader immediately knows gazelle will ignore it.